### PR TITLE
Translate the CNCF.en.md from CNCF.cn.md

### LINKED ISSUE #6070: [Translation] Translate technical articles concerning Apache ShardingSphere
Hi, community,

This issue is to call for volunteers to **translate some technical articles from Chinese into English**. Anyone who enjoys translation or wants to make some contributions to open source community is warmly welcomed! 👏 

### Competencies needed
1. English translation skill
2. Basically, know what ShardingSphere is doing

### Article list
- [x] 1. [Start ElasticJob off with new goal](https://mp.weixin.qq.com/s/XJFj_vOj3bR6HRQUzy1ikg) --> @rudy2steiner
- [x] 2. [How to construct the distributed database](https://mp.weixin.qq.com/s/YSoX0Apq6vQeu8TU_53EAw) --> @EasonFeng5870
- [x] 3. [Apache ShardingSphere is included in CNCF Landscape](https://mp.weixin.qq.com/s/hAxON1CMYvgFf6sFIHtDHA) --> @mlw12345678
- [x] 4. [The mixed open-source distributed transaction solution](https://mp.weixin.qq.com/s/fYESyuw7O4u7wB-tSWlhyg) --> @mlw12345678
- [x] 5. [Would you like to become an Apache committer](https://mp.weixin.qq.com/s/sFqOz10KTk23IhmmDTy67g) --> @mlw12345678
- [x] 6. [How automatic executor of ShardingSphere works](https://mp.weixin.qq.com/s/hIKZPJDfGfeduW4frwVN_w)
- [x] 7. [How to merge the child resultsets](https://mp.weixin.qq.com/s/5__SGXEdvGqkVhpnrW6zug)
- [ ] 8. [The quick explanation of ShardingSphere transaction module](https://mp.weixin.qq.com/s/0LtqRTL8HewUAdZCk8LRLg)
- [x] 9. [What is the ShardingProxy](https://mp.weixin.qq.com/s/bTX0n9tH4x8eue19b7rdGw) --> @wcsoft 

### How to submit your translation
1. Add your translation to `xxx.en.md` in [blog](https://github.com/apache/shardingsphere/tree/master/docs/blog/content/material).

2. Raise a PR for review.

### Notice
1. All the Chinese articles named `xxx.cn.md` could be found in the same directory of Your translation as well. 
2. If you want o find a partner to work together for these long articles, please be free to give `recruit comment` below. 😉 
3. You can give a brief translation for these author profiles in the articles. Like just translate their names and titles.

If you want to have a try, **please comment here with** `Article number and your estimated deadline`. 

Besides, **any feedback and comments are welcomed as well**! 😃

### PR DESCRIPTION
**Pull request by mlw12345678**
Fixes #6070 No3

Changes proposed in this pull request:
- docs/blog/content/material/CNCF.en.md

image

### CODE REVIEW

**Comment by tristaZero** [comment]
@mlw12345678 Very appreciated your rapid work! I will review this PR ASAP.

File: docs/blog/content/material/CNCF.en.md [hunk 1]
```diff
@@ -4,4 +4,32 @@ weight = 5
 chapter = true
 +++
 
-## TODO
+## Apache ShardingSphere is included in CNCF Landscape
+
+### CNCF
+
+(Cloud Native Computing Foundation)，is a cloud native computing open source software foundation founded by Google. It is committed to the popularization and sustainable development of Cloud Native technology. The representative cloud-native technologies are containers, microservices, microservice governance, and declarative APIs, etc.
```
Comment (tuohai666) [inline docs/blog/content/material/CNCF.en.md:11]: Miss one sentence.

**Comment by tuohai666** [comment]
@mlw12345678 Ping

File: docs/blog/content/material/CNCF.en.md (hunk 1 above)
Comment (mlw12345678) [reply docs/blog/content/material/CNCF.en.md:11]: I will fix it now!

Comment (mlw12345678) [reply docs/blog/content/material/CNCF.en.md]: Done , any other questions ?

**Pull request merged**